### PR TITLE
Discretize-then-project: Inconsistent Flux Method

### PR DESCRIPTION
**Pull request by ksstar**
- Full order model (FOM): Forward Euler - Inconsistent Flux Method (IFM) + Consistent flux Method
- Galerkin projection of the FOM with IFM including the projection of the boundary vectors (no penalty of lifting function method required)
- Reduced order model: Inconsistent Flux Method
- CFD tutorial 19: Lid driven cavity - Inconsistent Flux Methods

### CODE REVIEW

File: src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.H [hunk 1]
```diff
@@ -87,9 +87,12 @@ class steadyNS: public reductionProblem
 
         /// List of pointers used to form the velocity snapshots matrix
         PtrList<volVectorField> Ufield;
-
-        /// List of pointers used to form the supremizer snapshots matrix
-        PtrList<volVectorField> supfield;
+	
+	/// List of pointers used to form the supremizer snapshots matrix
+	PtrList<volVectorField> supfield;
```
Comment (giovastabile) [inline src/ITHACA_FOMPROBLEMS/steadyNS/steadyNS.H]: check indentation

File: src/ITHACA_ROMPROBLEMS/ReducedProblem/ReducedProblem.H [hunk 2]
```diff
@@ -93,7 +93,8 @@ class reducedProblem
         Eigen::MatrixXd solveLinearSys(List<Eigen::MatrixXd> LinSys, Eigen::MatrixXd x,
                                        Eigen::VectorXd& residual, const Eigen::MatrixXd& bc = Eigen::MatrixXd::Zero(0,
                                                0), const std::string solverType = "fullPivLu");
-        ///
+
+	///
```
Comment (giovastabile) [inline src/ITHACA_ROMPROBLEMS/ReducedProblem/ReducedProblem.H]: also here check indentation

**Comment by ksstar** [comment]
Ready for review

**Comment by ksstar** [comment]
Finished. Ready for review.

**Comment by ksstar** [comment]
All checks have passed. Ready to be merged if there are no further comments.

**Pull request merged**